frontend/packages/dev-console/src/utils/__tests__/pipeline-augment.spec.ts

```diff
@@ -274,3 +275,27 @@ describe('PipelineAugment test successfully determine Task type', () => {
     expect(model).toBe(ClusterTaskModel);
   });
 });
+
+describe('Pipeline exists test to determine whether a pipeline is linked to a pipelinerun', () => {
+  it('expect to return true if pipelineref is available in pipelinerun spec', () => {
+    const pipelineRunWithPipelineRef =
+      pipelineTestData[PipelineExampleNames.SIMPLE_PIPELINE].pipelineRuns[DataState.SUCCESS];
+
+    const pipelineFound = pipelineRefExists(pipelineRunWithPipelineRef);
+    expect(pipelineFound).toBeTruthy();
+  });
+
+  it('expect to return false if pipelineref is missing in pipelinerun spec', () => {
+    const pipelineRunWithoutPipelineRef = {
+      ...pipelineTestData[PipelineExampleNames.SIMPLE_PIPELINE].pipelineRuns[DataState.SUCCESS],
+      spec: {
+        ...pipelineTestData[PipelineExampleNames.SIMPLE_PIPELINE].pipelineRuns[DataState.SUCCESS]
+          .spec,
+        pipelineRef: undefined,
+      },
+    };
```

Comment:
Since we don't actually support the pipelineSpec with this change... we can keep this test as just omit the `pipelineRef`. Which can be simplified if we just use lodash to omit the field. 

```suggestion
    const pipelineRunWithoutPipelineRef = _.omit(
      pipelineTestData[PipelineExampleNames.SIMPLE_PIPELINE].pipelineRuns[DataState.SUCCESS],
      'spec.pipelineRef',
    );
```

Reply:
updated it.